kuma/landing/jinja2/landing/homepage.html

```diff
@@ -82,10 +82,8 @@ <h2 class="offscreen">{{ _('Web Technologies') }}</h2>
         <h3 class="zones"><i aria-hidden="true" class="icon-suitcase"></i><span>{{ _('Mozilla<br> Docs') }}</span></h3>
         <ul>
           <li><a href="{{ wiki_url('Mozilla/Firefox') }}">{{ _('Firefox') }}</a></li>
-          <li><a href="{{ wiki_url('Mozilla/B2G_OS') }}">{{ _('B2G OS') }}</a></li>
-          <li><a href="{{ wiki_url('Mozilla/Marketplace') }}">{{ _('Firefox Marketplace') }}</a></li>
-          <li><a href="{{ wiki_url('Apps') }}">{{ _('App Center') }}</a></li>
           <li><a href="{{ wiki_url('Mozilla/Add-ons') }}?menu">{{ _('Add-ons') }}</a></li>
+          <li><a href="{{ wiki_url('Mozilla/Add-ons/WebExtensions') }}">{{ _('WebExtensions') }}</a></li>
```

Comment:
Same question, but I see that it is styled as ``WebExtensions`` on the documentation page too. 👽